### PR TITLE
Adding SuperSlicer

### PR DESCRIPTION
**Pull request by MarioGK**
Used the prusaslice.json as a base to add the superslicer.json as the later is a fork of the former.
Used checkver.ps1 to check for hash and update url
It only has a 64 bit version no win32.

### CODE REVIEW

**Comment by MarioGK** [comment]
It is fully installing now you can use https://github.com/MarioGK/Extras to test its installationg

File: bucket/superslicer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "64bit": {
+            "url": "https://github.com/supermerill/SuperSlicer/releases/download/2.3.57.6/SuperSlicer_2.3.57.6_win64_211118.zip",
+            "hash": "a4ef771b1bc73e7e00b3bab413b22b339e624499e0d6df4f6e4d75e63b08ab16",
+            "extract_dir": "SuperSlicer_2.3.57.6_win64_211118"
+        }
+    },
+    "bin": [
+        "superslicer.exe",
+        "superslicer_console.exe"
+    ],
```
Comment (rashil2000) [inline bucket/superslicer.json:16]:
```suggestion
    "bin": "superslicer_console.exe",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by MarioGK** [comment]
@rashil2000 Made the requested changes, maybe i should update the PrusaSlicer one as well? because it has both .exes on the bin there as well

**Comment by rashil2000** [comment]
Sure

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**